### DISCUSSION TITLE
Dreams about the ppg bug tracker thread

### DISCUSSION THREAD

**Discussion by ghost** [studio-minus/ppg-bugs-and-requests#2463] [General]
i had a dream once, some fella on the discussion board requested a "too complex" tag that would be used for ideas that are too complex. that is basically rejected feature but with extra steps

so i made this thread in case someone else had dreams about the ppg bug tracker

rules are dont be a dunce

**Comment by TeslaCoil9001** [comment]
I had a dream about a world where everyone used the template

**Reply by ghost** [reply]
**Comment description** 
use the gdam comment template!!! 

**Comment purpose**
To remind to use the template

**Additional notes**
Not an offensive personal attack, just a small joke
